NextcloudTalk/NCRoom.m

```diff
@@ -68,6 +68,9 @@ + (instancetype)roomWithDictionary:(NSDictionary *)roomDict
     room.hasCall = [[roomDict objectForKey:@"hasCall"] boolValue];
     room.canLeaveConversation = [[roomDict objectForKey:@"canLeaveConversation"] boolValue];
     room.canDeleteConversation = [[roomDict objectForKey:@"canDeleteConversation"] boolValue];
+    room.status = ([roomDict objectForKey:@"status"] != [NSNull null]) ? [[roomDict objectForKey:@"status"] stringValue] : @"";
```

Comment:
I'm not absolutely sure about this one. Do we need to check for `[NSNull null]` here? I would imagine that
`[roomDict objectForKey:@"status"] ? [[roomDict objectForKey:@"status"] stringValue] : @""`